aten/src/ATen/native/transformers/cuda/sdp_utils.h

```diff
@@ -538,13 +536,13 @@ inline bool check_head_dim_gt64_and_sm_ge86(sdp_params params, bool debug) {
   return true;
 }
 
-inline bool check_requires_grad_and_head_dim_gt64_and_sm_ge86(
+inline bool check_requires_grad_and_head_dim_gt64_and_sm_ge86_lt90(
```

Comment:
I am landing https://github.com/pytorch/pytorch/pull/99948 which will cause merge conflicts so I would wait until this before tyring to land